docs/config.html

```diff
@@ -484,6 +485,65 @@ <h2 id="build-env" class="title-2">[BuildEnv]</h2>
   </p>
 </section>
 
+<section class="mt4">
+  <h2 id="sandbox" class="title-2">[Sandbox]</h2>
+  <ul class="bulleted-list">
+    <li>
+      <div>
+        <h3 class="mt1 f6 lh-title">
+          Tool
+        </h3>
+
+        <p>
+          The binary to use for sandboxing build/test actions. Typically this
+          is <code>please_sandbox</code>.<br />
+          It receives the program to execute and all its arguments in its <code>argv</code>,
+          it is expected to do whatever it needs to then <code>exec</code> the real action.
+        </p>
+      </div>
+    </li>
+    <li>
+      <div>
+        <h3 class="mt1 f6 lh-title">
+          Build <span class="normal">(bool)</span>
+        </h3>
+
+        <p>
+          Activates sandboxing for individual build actions, which isolates them
+          from network access, IPC and the repo-local filesystem. Currently
+          only works on Linux. Defaults to <code class="code">False</code>.
+        </p>
+      </div>
+    </li>
+    <li>
+      <div>
+        <h3 class="mt1 f6 lh-title">
+          Test <span class="normal">(bool)</span>
+        </h3>
+
+        <p>
+          Activates sandboxing for individual test actions, which isolates them
```

Comment:
What's the significance of individual here (and above)? Maybe just "Activates sandboxing for test actions,"? Or am I missing something?

Reply:
It's been copied & pasted a bit now, but I imagine what I was originally thinking was that it sandboxes them from one another (as opposed to sandboxing all build actions together in one place).
I'll remove it, it's not really adding anything.